src/app/App.js

```diff
@@ -1,19 +1,36 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useMemo } from "react";
 import { ThemeProvider, CssBaseline, Container, Box } from "@material-ui/core";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
 import { useSelector } from "react-redux";
 import ReactGa from "react-ga";
-import theme from "./theme";
-import Header from "../features/landing/header/Header";
+import theme from "./theme/theme";
 import BlurbSection from "../features/landing/blurbs/BlurbSection";
 import Bubbles from "../features/visualizations/bubbles/BubblesVis";
 import Cases from "../features/visualizations/cases/CasesVis";
 import Air from "../features/visualizations/air/AirVis";
 import RacingData from "../features/visualizations/Racing-Chart/RacingData";
 import BeforeFooter from "../features/landing/footer/BeforeFooter";
 import Footer from "../features/landing/footer/Footer";
-import ErrorBoundary from "./error/ErrorBoundary";
+import ColorModeToggle from "./theme/ColorMode";
+import Info from "../features/landing/header/Info";
+import Globe from "../features/landing/header/Globe";
+import { ReactComponent as Logo } from "../features/landing/header/logo.svg";
 
 const App = () => {
+  const { darkMode } = useSelector((state) => state.themeReducer);
```

Comment:
Great idea. This made me excited! my computer and phone is always in dark mode lol